src/main/java/io/zeebe/monitor/zeebe/ZeebeKafkaStreams.java

```diff
@@ -0,0 +1,100 @@
+package io.zeebe.monitor.zeebe;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.zeebe.monitor.rest.dto.GenericKafkaRecord;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.kstream.Consumed;
+import org.apache.kafka.streams.kstream.KStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ZeebeKafkaStreams {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+    @Autowired
+    public StreamsBuilder kStreamBuilder;
```

Comment:
var name is very misleading. You're using `StreamsBuilder` class, but naming it with deprecated `kStreamBuilder`